tests/testthat/dalys_parameters.csv

```diff
@@ -0,0 +1,69 @@
+disease,modelling_group,vimc_dalys,burden_outcome_id,code,sub_condition,proportion,average_duration,period,disability_weight
```

Comment:
should this go with the test data perhaps?

Reply:
Move this to `inst/dalys_parameters.R`, then see suggestions in the R file

Reply:
Done.